examples/Shifting/Shifting.ino

```diff
@@ -0,0 +1,56 @@
+/**********************************************
+ * AntPlus shifting shifter example
+ *
+ * Deliver data of a shifting sensor
+ * to a display via serial port
+ *
+ * Author Bernd Wok�ck
+ *        based on the work of Curtis Malainey
+ **********************************************/
+#include <Arduino.h>
+#include "ANT.h"
+#include "ANTPLUS.h"
+
+#define BAUD_RATE 9600
+#define CHANNEL_1 0
```

Comment:
We also could name it CHANNEL_A. Implying a strict assisgnment between constant and value is not the philosphy of defines. Playing around with the channels values made me discover the library bug, that it only worked with channel 0. For ANT the channel number does not play a role, it is just a resource, that must be assigned. Feel free, to rename it as you like.

Reply:
I have no issues with an alternative name, number is simply the construct employed through the API, but I do want to avoid confusion. I also like using defines to make things a bit more verbose.